test/e2e/suite/certificates/additionaloutputformats.go

```diff
@@ -49,9 +49,7 @@ var _ = framework.CertManagerDescribe("Certificate AdditionalCertificateOutputFo
 	)
 
 	createCertificate := func(f *framework.Framework, aof []cmapi.CertificateAdditionalOutputFormat) *cmapi.Certificate {
-		if !utilfeature.DefaultFeatureGate.Enabled(feature.AdditionalCertificateOutputFormats) {
-			framework.Skipf("feature gate %q is not enabled, skipping test", feature.AdditionalCertificateOutputFormats)
-		}
+		framework.RequireFeatureGate(f, utilfeature.DefaultFeatureGate, feature.AdditionalCertificateOutputFormats)
```

Comment:
🤔 how does any of this code work? & how did it work before? Feature flags aren't passed to the e2e framework so how is this behaviour actually being toggled on during e2e?

Reply:
aha, I see here: https://github.com/cert-manager/cert-manager/blob/0860a4141b99412ae32c0d13d5939fbbe3d82159/devel/run-e2e.sh#L74